data/actionsmap/yunohost.yml

```diff
@@ -481,6 +481,21 @@ domain:
                             - !!str ^[0-9]+$
                             - "pattern_positive_number"
 
+        ### domain_maindomain()
+        main-domain:
+            action_help: Check the current main domain, or change it
+            deprecated_alias:
+                - maindomain
```

Comment:
Will this really work considering that it's not in the same category ? Or am I mistaking what this does ?

Reply:
This one will work for when you write `yunohost domain maindomain`, for `yunohost tools maindomain` it's the `logger.warning` that will handle it.